app/Console/Commands/BulkReviewPosts.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Rogue\Console\Commands;
+
+use Rogue\Models\Post;
+use Illuminate\Console\Command;
+use Rogue\Managers\PostManager;
+
+class BulkReviewPosts extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'rogue:bulkreviewposts {campaign} {oldStatus} {newStatus} {--type= : Filter by the type of post} {--logfreq=1000} {--log} {--tag=* : Tag(s) slug to tag post by. e.g. hide-in-gallery}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Bulk review and tag posts constrained by the provided parameters';
+
+    /**
+     * The Post Manager instance.
+     *
+     * @var Rogue\Managers\PostManager
+     */
+    protected $posts;
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct(PostManager $posts)
+    {
+        parent::__construct();
+
+        $this->posts = $posts;
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        info('rogue:bulkreviewposts: Starting to bulk accept posts!');
+
+        $postType = $this->option('type') ?? null;
+        $logfreq = $this->option('logfreq');
+        $log = $this->option('log');
+        $tags = $this->option('tag') ?? null;
+
+        $query = (new Post)->newQuery();
```

Comment:
I think you can simplify this by querying the model directly.

```
$postQuery = Post::where([
    ['campaign_id', '=', $this->argument('campaign')],
    ['status', '<>',  $this->argument('oldStatus'),
]);

if ($postType !== null) {
   $postQuery  = $query->where('type', $postType);
}

$posts  = $postQuery->get();
```

Reply:
ooo so much cleaner @sbsmith86 ! I didn't know you could do both `Post::where` and `$query->where` together! 

One q - I've never seen `<>` before and I think it means `!=` - in this case, would we want the below instead? Not sure if I'm not understanding something!

```
$postQuery = Post::where([
    ['campaign_id', '=', $this->argument('campaign')],
    ['status', '=',  $this->argument('oldStatus'),
]);

if ($postType !== null) {
   $postQuery  = $query->where('type', $postType);
}

$posts  = $postQuery->get();
```

Reply:
oh yes yes, i was copying from the example and missed that!